src/components/NoteDetails/NoteDetailsStyles.js

```diff
@@ -26,7 +27,6 @@ export const StyledNoteNameEditor = styled(DefaultTextarea)`
   font-size: 20px;
   font-weight: 500;
   flex-grow: 0;
-  margin-top: -24px; // override status message 'Saved...'
```

Comment:
Please add this line back, as it breaks the styles:
Before your change:
![image](https://user-images.githubusercontent.com/6686679/116073474-21ab1780-a699-11eb-8941-64d3f18e4e1f.png)

After your change:
![image](https://user-images.githubusercontent.com/6686679/116073453-19eb7300-a699-11eb-8a9d-41f34abcff4b.png)

Notice a black block between the topbar and the note name editor